pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -80,10 +80,23 @@ func NewOvirtMapper(vm *ovirtsdk.Vm, mappings *v2vv1alpha1.OvirtMappings, creds
 }
 
 // CreateEmptyVM creates empty virtual machine definition
-func (o *OvirtMapper) CreateEmptyVM() *kubevirtv1.VirtualMachine {
+func (o *OvirtMapper) CreateEmptyVM(vmName *string) *kubevirtv1.VirtualMachine {
 	return &kubevirtv1.VirtualMachine{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{
+				"app":                                *vmName,
+				"flavor.template.kubevirt.io/Custom": "true",
+			},
+		},
 		Spec: kubevirtv1.VirtualMachineSpec{
 			Template: &kubevirtv1.VirtualMachineInstanceTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"kubevirt.io/domain":                 *vmName,
+						"flavor.template.kubevirt.io/Custom": "true",
```

Comment:
just a quick note, this convention is used only by the UI at the moment. Maybe we should standardize it and have it lowercase?

`flavor.template.kubevirt.io/custom`

Reply:
+1 according to k8s convention, annotations/labels should be in lowercase

Reply:
ack